oci/private/tarball_run.sh.tpl

```diff
@@ -2,13 +2,15 @@
 set -o pipefail -o errexit -o nounset
 
 readonly IMAGE="{{image_path}}"
-if command -v docker &> /dev/null; then
+if [ -e "{{command}}" ]; then
+    CONTAINER_CLI="{{command}}"
+elif command -v docker &> /dev/null; then
     CONTAINER_CLI="docker"
 elif command -v podman &> /dev/null; then
     CONTAINER_CLI="podman"
 else
     echo >&2 "Neither docker or podman could be found."
-    echo >&2 "If you wish to use another container runtime, please comment on https://github.com/bazel-contrib/rules_oci/issues/295."
+    echo >&2 "If you wish to use another container runtime, you can pass command to oci_tarball(...)."
```

Comment:
```suggestion
    echo >&2 "If you wish to use another container runtime, you can pass an executable to the `command` attribute."
```

Reply:
Good suggestion, although I think the backticks are dangerous in a shell:

```
$ echo "If you wish to use another container runtime, you can pass an executable to the `command` attribute."
If you wish to use another container runtime, you can pass an executable to the  attribute.
```

Will accept wording but change to single quotes.